PWGHF/HFC/TableProducer/correlatorDstarHadrons.cxx

```diff
@@ -0,0 +1,285 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file correlatorDstarHadron.cxx
+/// \author Deependra Sharma <deependra.sharma@cern.ch>, IITB
+/// \author Fabrizio Grosa <fabrizio.grosa@cern.ch>, CERN
+
+// O2
+#include "CommonConstants/PhysicsConstants.h"
+#include "Framework/ASoAHelpers.h"
```

Comment:
Where do you use `ASoAHelpers`?

Reply:
As per documentation, "You can use helper functions from ASoAHelpers.h to get combinations of elements (collisions, tracks, …). "
I have used ```soa::CombinationsFullIndexPolicy(...)``` in function ```processDataWithMixedEvent(...)```.